browser/ui/webui/private_new_tab_page/brave_private_new_tab_page_handler.cc

```diff
@@ -93,6 +94,20 @@ void BravePrivateNewTabPageHandler::GetIsTorConnected(
   std::move(callback).Run(is_connected);
 }
 
+void BravePrivateNewTabPageHandler::GetIsTorDisabled(
+    GetIsTorDisabledCallback callback) {
+  bool is_disabled = false;
+#if BUILDFLAG(ENABLE_TOR)
+  if (profile_) {
```

Comment:
Q. When can `profile_` be invalid? If it's for test, would it be worth adding `CHECK_IS_TEST()`?

```
if (profile_) {
  ...
} else {
  CHECK_IS_TEST();
}
```